mathesar/tests/api/test_user_api.py

```diff
@@ -61,10 +61,11 @@ def test_user_password_reset_non_superuser(client_bob, user_bob):
 
 
 def test_user_password_change(client_bob, user_bob):
```

Comment:
Can you add a test similar to this one, which tests the error behavior of a user trying to enter an invalid password?